.github/scripts/update-manifests-tags.js

```diff
@@ -0,0 +1,67 @@
+const { parseManifestFile, updateManifestFile } = require('./manifest-utils');
+
+/**
+ * Update component manifest references in get_all_manifests.sh
+ * Reads environment variables exported by get-release-branches.js
+ */
+
+module.exports = () => {
+    const manifestFile = 'get_all_manifests.sh';
+
+    console.log('Updating component branches/tags...');
+
+    const manifestComponents = parseManifestFile(manifestFile);
+
+    const specPrefix = 'component_spec_';
+
+    const updates = new Map();
+
+    for (const [key, value] of Object.entries(process.env)) {
+        if (key.startsWith(specPrefix)) {
```

Comment:
To avoid that all code is nested, it could be

```suggestion
        if (!key.startsWith(specPrefix)) {
            continue
        }
        ...
```